tests/fetch_test.go

```diff
@@ -14,7 +14,13 @@ import (
 
 // pathForFetchCmd constructs the .mp4 path based on the "from" field in the command.
 func pathForFetchCmd(fromTimestamp string) string {
-	return "/tmp/" + videoStoreComponentName + "_" + fromTimestamp + ".mp4"
+	// Parse the datetime format timestamp
+	t, err := time.Parse("2006-01-02_15-04-05", fromTimestamp)
+	if err != nil {
+		return "/tmp/" + videoStoreComponentName + "_" + fromTimestamp + ".mp4"
+	}
+	// Convert to Unix timestamp
```

Comment:
Is it documented that when `time.Parse` fails the returned timestamp is valid?

I don't think this is safe behavior.

Does this actually work?

Reply:
>I don't think this is safe behavior.

You're right. This branch is never hit. It should be `err == nil`

Reply:
Removed complexity after [realizing](https://viaminc.slack.com/archives/C06KBQEQDNC/p1742828601511129?thread_ts=1742825554.647749&cid=C06KBQEQDNC)